app/javascript/pages/country/widgets/widget-tree-cover-gain/widget-tree-cover-gain-reducers.js

```diff
@@ -29,21 +45,20 @@ const setTreeCoverGainIsLoading = (state, { payload }) => ({
 const setTreeCoverGainValues = (state, { payload }) => ({
   ...state,
   isLoading: false,
-  totalAmount: payload.totalAmount,
-  percentage: payload.percentage
+  gain: payload.gain,
+  treeExtent: payload.treeExtent
 });
 
-const setTreeCoverGainSettingsLocation = (state, { payload }) => ({
+const setTreeCoverGainSettingsIndicator = (state, { payload }) => ({
   ...state,
   settings: {
     ...state.settings,
-    location: payload.value,
-    locationLabel: payload.label
+    indicator: payload
```

Comment:
When you have finished the helper, lets make sure this value (not label) is correct (i.e. 'gadm28' or 'plantations')